src/watchmaker/workers/salt.py

```diff
@@ -631,20 +632,22 @@ def process_states(self, states, exclude):
             return
 
         cmds = []
-        states = states.lower().split(',')
```

Comment:
This _should_ already be case insensitve... 

```
$ python
>>> 'highstate' in 'Highstate,foo,bar'.lower().split(',')
True
```

Reply:
Oh I see, that _other_ state names are not case insensitive?

Reply:
> Oh I see, that _other_ state names are not case insensitive?

Yup.  That's the issue.

Reply:
```suggestion
        states = states.split(',')
```

Reply:
```suggestion
        states = states.split(',')
```